src/components/OrdersWidget/index.tsx

```diff
@@ -140,6 +141,9 @@ const OrdersWidget: React.FC = () => {
     [displayedOrders],
   )
 
+  // Sort validUntil
+  const { sortTopic, setSortTopic } = useSortByTopic<AuctionElement>(displayedOrders, 'validUntil')
```

Comment:
nit: move `validUntil` to a constant i.e. `DEFAULT_SORT_BY`

Also, what about creating a type `type OrderSortByField = 'validUntil' | '...'`
Also, I would add a 2nd generic param for the sort by topic. Instead of string, is nicer to use `OrderSortByField`